src/posix/platform/radio_spinel.cpp

```diff
@@ -946,6 +946,20 @@ int8_t RadioSpinel::GetRssi(void)
     return rssi;
 }
 
+const char *RadioSpinel::GetVersion(void)
```

Comment:
Fill version just once in `RadioSpinel::Init()`?

Reply:
Was considering this actually but felt safer to read it from radio directly (just in case we let RCP get updated).

Reply:
If it’s updated, it should be reinitialised, shouldn’t it?

Reply:
I am ok with moving this to `Init()`. I was thinking if there is a way to avoid having the member variable buffer to store the version but cannot find any easy way around it.

Reply:
Yes, the buffer must be provided either by RadioSpinel or the caller.